website/content/v1.12/reference/cli.md

```diff
@@ -149,9 +149,49 @@ talosctl cluster create docker [flags]
 ### Options inherited from parent commands
 
 ```
-      --name string          the name of the cluster (default "talos-default")
-      --provisioner string   Talos cluster provisioner to use (default "docker")
-      --state string         directory path to store cluster state (default "/home/user/.talos/clusters")
+      --name string    the name of the cluster (default "talos-default")
+      --state string   directory path to store cluster state (default "/home/user/.talos/clusters")
+```
+
+### SEE ALSO
+
+* [talosctl cluster create](#talosctl-cluster-create)	 - Creates a local qemu based cluster for Talos development
+
+## talosctl cluster create qemu
+
+Create a local Qemu based kubernetes cluster
+
+```
+talosctl cluster create qemu [flags]
+```
+
+### Options
+
+```
+      --cidr string                              CIDR of the cluster network (default "10.5.0.0/24")
+      --config-patch stringArray                 patch generated machineconfigs (applied to all node types), use @file to read a patch from file
+      --config-patch-controlplanes stringArray   patch generated machineconfigs (applied to 'controlplane' types)
+      --config-patch-workers stringArray         patch generated machineconfigs (applied to 'worker' type)
+      --controlplanes int                        the number of controlplanes to create (default 1)
+      --cpus-controlplanes string                the share of CPUs as fraction (each control plane/VM) (default "2.0")
+      --cpus-workers string                      the share of CPUs as fraction (each worker/VM) (default "2.0")
+      --disks strings                            list of disks to create in format "<driver1>:<size1>" (size is specified in megabytes) (disks after the first one are added only to worker machines) (default [virtio:6144])
```

Comment:
use human readable and bump the default, probably allocate another disk by default!

```suggestion
      --disks strings                            list of disks to create in format "<driver1>:<size1>" (size is specified in megabytes) (disks after the first one are added only to worker machines) (default [virtio:10GB])
```

Reply:
disable pre-allocate on files backing disks